storage/scylladb_client.h

```diff
@@ -13,10 +13,12 @@ extern "C" {
 
 #include "scylladb_utils.h"
 
+#define DB_UUID_STRING_LENGTH CASS_UUID_STRING_LENGTH
 typedef struct {
   CassCluster* cluster;
   CassSession* session;
   char* host;
+  CassUuidGen* uuid_gen;
```

Comment:
Would it be better to rename to `uuid` instead?

Reply:
I am not familiar with `CassUuidGen*`. Which did it refer to?

Reply:
It is a UUID generator. 

Use `cass_uuid_gen_new()` to create the object.

From header(cassandra.h) comment :
> This object is thread-safe. It is best practice to create and reuse a single object per application.

> If unique node information (IP address) is unable to be determined then random node information will be generated.

Also described in [DataStax](https://docs.datastax.com/en/developer/cpp-driver/2.14/api/struct.CassUuidGen/).

Reply:
Let's have inlined comment for `uuid_gen`.